source/kraken/maintenance_worker.h

```diff
@@ -40,12 +40,16 @@ www.navitia.io
 
 namespace navitia {
 
+class Metrics;
```

Comment:
Or you can just add the `#include "metrics.h` here, instead of using _forward declaration._  
As you wish.

Reply:
forward declare in .h limit compilation times.